utils/cards.js

```diff
@@ -0,0 +1,24 @@
+export const cards = [
```

Comment:
Ah yes I'm so glad this finally happened! This is 100% better outside of the pages/components